PocketKit/Sources/Analytics/AppEvents/Home.swift

```diff
@@ -0,0 +1,229 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+public extension Events {
+    struct Home {}
+}
+
+public extension Events.Home {
+    /**
+     Fired when a card in the `Recent Saves` section scrolls into view
+     */
+    static func RecentSavesCardImpression(url: URL!, positionInList: Int) -> Impression {
```

Comment:
are we okay with forcing this type? I have thought of forcing was bad due to crashes, but happy to change this mindset if we see it fits this codebase better

Reply:
will update. idk why i did this.